lib/process-services/src/lib/task-comments/task-comments.component.spec.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TaskCommentsComponent } from './task-comments.component';
+
+describe('TaskCommentsComponent', () => {
+    let component: TaskCommentsComponent;
+    let fixture: ComponentFixture<TaskCommentsComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [TaskCommentsComponent]
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(TaskCommentsComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
```

Comment:
this is a dummy test and does not make much sense, probably needs proper tests here